package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hof-bootstrap",
   "description": "A bootstrap for HOF projects",
-  "version": "6.0.2",
+  "version": "7.0.0",
```

Comment:
Because this is _only_ updating the version number in package.json, meaning package.json isn't in sync with npm or github. The person who merges the PR for changelog should update master and run `npm version major; git push origin master --tags; npm publish`